src/py42/modules/archive.py

```diff
@@ -106,3 +106,16 @@ def get_all_device_restore_history(self, days, device_id):
             that each contain a page of restore history.
         """
         return self._archive_client.get_all_restore_history(days, u"computerId", device_id)
+
+    def update_cold_storage_purge_date(self, archive_guid, purge_date):
+        """Updates the cold storage purge date for a specified archive.
+        REST Documentation <https://www.crashplan.com/apidocviewer/#ColdStorage-put>
+
+        Args:
+            archive_guid (str): The identification number of the archive that should be updated
+            purge_date (str): The date on which the archive should be purged in yyyy-MM-dd format
+
+        Returns:
+            :class: py42.response.Py42Response: the response from the ColdStorage API.
```

Comment:
Similar thing here -- to get the docs links to build, I believe you have to use tick marks like this: :class:\`py42.response.Py42Response`

Reply:
I think I've got it right now.

Reply:
The tic marks were not showing up in the diff so I went in edit mode to see and there was space between :class: and the class name, so that may have been a small issue. I just went ahead and removed the space just in case!